chaincode/tuple_aggregate.go

```diff
@@ -390,6 +390,18 @@ func logSuccessAggregate(db *LedgerDB, args []string) (o outputAggregatetuple, e
 		return
 	}
 
+	// send an compute plan event if this one is done
```

Comment:
1. Is there a reason why we don't send an event in the "failed" case?
2. If two tuples from the same CP get marked as failed,l we're going to generate two events. Is that intentional?

Reply:
For now we only send an event if there is a success for computeplan. It's purpose is to remove the `/local` directory in the different workers. In other cases (`Failed` and `Cancelled`), it should not be removed automatically.